x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/phases/cold_phase.tsx

```diff
@@ -101,68 +117,125 @@ export class ColdPhase extends PureComponent<Props> {
-                />
-
-                <EuiFlexGroup>
-                  <EuiFlexItem grow={false} style={{ maxWidth: 188 }}>
-                    <ErrableFormRow
-                      id={`${coldProperty}-${phaseProperty('freezeEnabled')}`}
-                      label={
-                        <Fragment>
-                          <FormattedMessage
-                            id="xpack.indexLifecycleMgmt.coldPhase.numberOfReplicasLabel"
-                            defaultMessage="Number of replicas"
-                          />
-                          <OptionalLabel />
-                        </Fragment>
-                      }
-                      isShowingErrors={isShowingErrors}
-                      errors={errors?.freezeEnabled}
-                      helpText={i18n.translate(
-                        'xpack.indexLifecycleMgmt.coldPhase.replicaCountHelpText',
-                        {
-                          defaultMessage: 'By default, the number of replicas remains the same.',
-                        }
-                      )}
-                    >
-                      <EuiFieldNumber
-                        id={`${coldProperty}-${phaseProperty('selectedReplicaCount')}`}
-                        value={phaseData.selectedReplicaCount}
-                        onChange={(e) => {
-                          setPhaseData(phaseProperty('selectedReplicaCount'), e.target.value);
-                        }}
-                        min={0}
-                      />
-                    </ErrableFormRow>
-                  </EuiFlexItem>
-                </EuiFlexGroup>
-              </Fragment>
-            ) : (
-              <div />
-            )}
-          </Fragment>
+          {phaseData.phaseEnabled ? (
+            <MinAgeInput<ColdPhaseInterface>
+              errors={errors}
+              phaseData={phaseData}
+              phase={coldProperty}
+              isShowingErrors={isShowingErrors}
+              setPhaseData={setPhaseData}
+              rolloverEnabled={hotPhaseRolloverEnabled}
+            />
+          ) : null}
         </EuiDescribedFormGroup>
         {phaseData.phaseEnabled ? (
           <Fragment>
+            {/* Data tier allocation section */}
+            <NodesDataProvider>
+              {(nodesData) => (
+                <EuiDescribedFormGroup
+                  title={
+                    <h3>
+                      {i18n.translate(
+                        'xpack.indexLifecycleMgmt.editPolicy.coldPhase.dataTierAllocationTitle',
+                        { defaultMessage: 'Data tier allocation' }
+                      )}
+                    </h3>
+                  }
+                  description={
+                    <>
+                      <FormattedMessage
+                        id="xpack.indexLifecycleMgmt.editPolicy.coldPhase.dataTierAllocationDescription"
+                        defaultMessage="Allocate cold phase data to nodes in the cluster."
```

Comment:
I'm not sure of the context here, but this seems a bit vague. Also, I'm not sure we want to introduce "cold phase data" as a term, I think it can just be "cold data".

Reply:
This is intended as the explanation text to accompany the title "Data tier allocation"

<img width="1048" alt="Screenshot 2020-09-09 at 16 07 01" src="https://user-images.githubusercontent.com/8155004/92609205-9a01cb80-f2b6-11ea-87e7-658759d55523.png">